### PR TITLE
Gnuradio 3.8.1.0

### PR DESCRIPTION
**Pull request by LoPablo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gnuradio.rb [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+
   desc "SDK providing the signal processing runtime and processing blocks"
   homepage "https://gnuradio.org/"
-  url "https://gnuradio.org/releases/gnuradio/gnuradio-3.7.13.4.tar.gz"
-  sha256 "c536c268b1e9c24f1206bbc881a5819ac46e662f4e8beaded6f3f441d3502f0d"
-  revision 14
+  url "https://github.com/gnuradio/gnuradio/releases/download/v3.8.1.0/gnuradio-3.8.1.0.tar.gz"
+  sha256 "e15311e7da9fe2bb790cc36321d7eb2d93b9dfa0c1552fa5d534dd99d22873be"
   head "https://github.com/gnuradio/gnuradio.git"
+  revision 15
```
Comment (jonchang) [inline Formula/gnuradio.rb:10]:
```suggestion
  head "https://github.com/gnuradio/gnuradio.git"
```

File: Formula/gnuradio.rb [hunk 2]
```diff
@@ -15,82 +17,87 @@ class Gnuradio < Formula
...
+  end
 
-    %w[Markdown Cheetah MarkupSafe Mako six].each do |r|
-      resource(r).stage do
-        system "python", *Language::Python.setup_install_args(libexec/"vendor")
-      end
-    end
+  def install
+    ENV.cxx11
+    ENV.prepend_path "PATH", "#{Formula["qt"].opt_bin}"
```
Comment (jonchang) [inline Formula/gnuradio.rb:91]:
```suggestion
    ENV.prepend_path "PATH", Formula["qt"].opt_bin
```

**Review by jonchang — CHANGES_REQUESTED** [review]
Looks good so far!

**Comment by iMichka** [comment]
See my comments and the comments below: https://github.com/Homebrew/homebrew-core/pull/49733#issuecomment-643090306

**Pull request closed without merging**